schema.sql

```diff
@@ -1,5 +1,10 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
+    id INT NOT NULL PRIMARY KEY,
```

Comment:
- [ ] [OPTIONAL] Kindly make the id column autoincremented, so you don't have to insert it manually.
```
id int IDENTITY(1,1)
```

Reply:
@gemmen29 Thank you for pointing this out, however, for the PostgreSQL version my OS is using I can only implement this by doing `id BIGSERIAL NOT NULL PRIMARY KEY` :+1: